docs/tools/terraform/reference/cookbook/kafka-mongodb-recipe.rst

```diff
@@ -0,0 +1,86 @@
+Apache Kafka® with MongoDB Source Connector
+===========================================
+
+The `Aiven Terraform Provider <https://registry.terraform.io/providers/aiven/aiven/latest/docs>`_ is a great choice for declaratively provisioning an Aiven for Apache Kafka® cluster with Kafka Connect enabled and the MongoDB source connector configured.
+
+Let's check out the following diagram to understand the setup.
+
+.. mermaid::
+
+   flowchart LR
+      id1[(MongoDB Database)]
+      subgraph Kafka Connect
+      MongoDB-Source-Connector
+      end
+      subgraph Apache Kafka
+      Topic
+      end
+      id1 --->|polls changes| MongoDB-Source-Connector -->|publishes changes| Topic
+
+Describe the setup
+------------------
+
+Here is the Terraform recipe that will spin up an Aiven for Apache Kafka service with Kafka Connect enabled. This recipe will also create and configure a MongoDB source connector. 
```

Comment:
Maybe we can add a link here to https://developer.aiven.io/docs/products/kafka/kafka-connect/howto/mongodb-poll-source-connector.html

Reply:
Thanks. Good point. I'm adding the link when I first mention MongoDB source connector, i.e. in the first paragraph.